querier_test.go

```diff
@@ -0,0 +1,35 @@
+package reform_test
+
+import (
+	"gopkg.in/reform.v1/dialects/mssql"
+	"gopkg.in/reform.v1/dialects/mysql"
+	"gopkg.in/reform.v1/dialects/postgresql"
+	"gopkg.in/reform.v1/dialects/sqlite3"
+	"gopkg.in/reform.v1/dialects/sqlserver"
+	. "gopkg.in/reform.v1/internal/test/models"
+)
+
+func (s *ReformSuite) TestQualifiedView() {
+	switch s.q.Dialect {
+	case postgresql.Dialect:
+		s.Equal(`"people"`, s.q.QualifiedView(PersonTable))
+		s.Equal(`"people"`, s.q.WithQualifiedViewName("ignored").QualifiedView(PersonTable))
+		s.Equal(`"legacy"."people"`, s.q.QualifiedView(LegacyPersonTable))
+		s.Equal(`"legacy"."people"`, s.q.WithQualifiedViewName("ignored").QualifiedView(LegacyPersonTable))
```

Comment:
invalid operation: s.q (variable of type *gopkg.in/reform.v1.Querier) has no field or method WithQualifiedViewName